sklearn/utils/estimator_checks.py

```diff
@@ -553,6 +554,45 @@ def check_sample_weights_list(name, estimator_orig):
         estimator.fit(X, y, sample_weight=sample_weight)
 
 
+@ignore_warnings(category=(DeprecationWarning, FutureWarning))
+def check_sample_weight_invariance(name, estimator_orig):
```

Comment:
It could be worth to add a one line comment regarding the purpose of the test

Reply:
Done!